### ISSUE TITLE
The code related to the pop-up window reports an error

### ISSUE THREAD

**Issue by wling-art** [plouvart/character-ai-bindings#1]
Traceback (most recent call last):
  File "C:\Users\wling\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\character_ai_bindings\chatroom.py", line 73, in __init__
    elem = WebDriverWait(self.driver, 30).until(
  File "C:\Users\wling\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\selenium\webdriver\support\wait.py", line 95, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
Backtrace:
        (No symbol) [0x0079DCE3]
        (No symbol) [0x007339D1]
        (No symbol) [0x00644DA8]
        (No symbol) [0x0067019F]
        (No symbol) [0x006703AB]
        (No symbol) [0x0069EE62]
        (No symbol) [0x0068AF14]
        (No symbol) [0x0069D57C]
        (No symbol) [0x0068ACC6]
        (No symbol) [0x00666F68]
        (No symbol) [0x006680CD]
        GetHandleVerifier [0x00A13832+2506274]
        GetHandleVerifier [0x00A49794+2727300]
        GetHandleVerifier [0x00A4E36C+2746716]
        GetHandleVerifier [0x00846690+617600]
        (No symbol) [0x0073C712]
        (No symbol) [0x00741FF8]
        (No symbol) [0x007420DB]
        (No symbol) [0x0074C63B]
        BaseThreadInitThunk [0x76328039+25]
        RtlInitializeExceptionChain [0x7730DEAB+107]
        RtlGetAppContainerNamedObjectPath [0x7730DE2D+237]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "e:\Users\wling\Desktop\code\python\chai.py", line 3, in <module>
    server = ChatRoomServer(
  File "C:\Users\wling\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\character_ai_bindings\chatroom.py", line 77, in __init__
    raise Exception("Couldn't close popup")
Exception: Couldn't close popup

**Comment by Unbewohnte** [comment]
I am experiencing the same issue. Getting rid of the whole pop up handling code and rebuilding the package (obviously) didn't help either as all further XPATH searches get invalid

**Comment by plouvart** [comment]
Hi there!
Probably character.ai changed some stuff on their page during the logging phase. (haven't logged in in a while)
I'm deep into some work for my company but I'll have a look at fixing that before the week end.
If you can't wait, you can try and visually see what's happening (wrong) by disabling headless mode for the chrome driver in the lib. That way you'll see what effectively prevents the script from going further.

**Comment by plouvart** [comment]
The accept button xpath location was changed.
I changed it. It should be fixed for now.
Report if it does not.

**Comment by Unbewohnte** [comment]
Wow, thank you for such a swift response and a fix !

Yes, now pop up does not cause any problems, account login works as intended as well.

Now it's stuck at `The AI is unresponsive!` exception at `ask_chatroom` function called through the instance of `ChatRoom`  because somehow chatbot's message `<div>` is not being detected. XPATH you've specified is the same as the path I've manually extracted so I wonder what the problem might be.

It's absolutely fine to take it slowly if you're busy, I'm not the one to change your schedule, look at it at your leisure. 

Thank you once again !

Should I open a new issue for that one ?

**Comment by plouvart** [comment]
Indeed I get that error now too. No need to open another issue.
The xpath changed on my end so I changed it. Although it now seems to work, my first attempts got timed-out. So maybe there is a behavior inconsistency? 
I suspect this might be related to the element showing up but somehow not triggering the "element located"-condition of selenium?
I wrapped this line in a loop for more attempts. That way, even if the first attempt times out, the second one may work.

Let me know if it works now!
